### PR TITLE
Searchpage group search functionality

### PR DESCRIPTION
**Pull request by squeakypickles33**
Next adding friends search results

### CODE REVIEW

**Comment by squeakypickles33** [comment]
meant 'trpc procedure' not 'router' in commit message

File: src/pages/search.tsx [hunk 1]
```diff
@@ -1,55 +1,48 @@
-import Head from "next/head";
-import Link from "next/link";
+import Head from 'next/head';
+import { useRouter } from 'next/router';
+import { api } from '@/utils/api';
+import { Link } from 'lucide-react';
```
Comment (burhan-syed) [inline src/pages/search.tsx]: oops you imported wrong link component. should be from next/link

**Review by burhan-syed — CHANGES_REQUESTED** [review]

File: src/pages/search.tsx (hunk 1 above)
Comment (squeakypickles33) [reply src/pages/search.tsx]: woops indeed! easy fix

File: src/components/Nav/Search.tsx [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
     <div>
       <search>
-        <form className="text-base text-base-content">
+        <form onSubmit={handleSearch} className="text-base text-base-content">
           <label htmlFor="search" className="sr-only">
             Search
           </label>
           <div className="join">
-            <div className="join-item btn rounded-l-full"><SearchIcon size={20} /></div>
+            <Link href='' onClick={handleSearch} className="join-item btn rounded-l-full"><SearchIcon size={20} /></Link>
```
Comment (burhan-syed) [inline src/components/Nav/Search.tsx:23]: this is a little odd, it should probably just be a button instead of a link.

**Review by burhan-syed — APPROVED** [review]

**Pull request merged**